### PR TITLE
[FEATURE] Added version no. of libraries to --version

### PR DESCRIPTION
**Pull request by mahalwal**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [X] I have read and understood the [contributors guide](https://github.com/CCExtractor/ccextractor/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that another pull request for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used CCExtractor.
- [X] I have used CCExtractor just a couple of times.
- [ ] I absolutely love CCExtractor, but have not contributed previously.
- [ ] I am an active contributor to CCExtractor.

---
#936 
Following libraries don't have any way to print their version numbers, so I just printed their names.
- zvbi
- nuklear
- libhash

FreeType has these macros FREETYPE_MAJOR, FREETYPE_MINOR, FREETYPE_PATCH which can be used, but it is throwing some errors which I was not able to fix.

### CODE REVIEW

File: src/lib_ccx/params.c [hunk 1]
```diff
@@ -953,6 +960,21 @@ void version(char *location) {
 	mprint("	Git commit: %s\n", GIT_COMMIT);
 	mprint("	Compilation date: %s\n", COMPILE_DATE);
 	mprint("	File SHA256: %s\n", hash);
+
+	mprint("Libraries used by CCExtractor\n");
+	mprint("	Tesseract Version: %s\n", (const char*)TessVersion());
```
Comment (canihavesomecoffee) [inline src/lib_ccx/params.c]: Did you take into account a version that has no Tesseract/Leptonica support included?

Comment (mahalwal) [reply src/lib_ccx/params.c]: Sorry, by 'a version' you mean of CCExtractor which has no Tesseract/Leptonica support?

Comment (canihavesomecoffee) [reply src/lib_ccx/params.c]: Yes, it's possible to compile CCExtractor without OCR support.

Comment (mahalwal) [reply src/lib_ccx/params.c]: Oh I'll wrap the two statements in #ifdef then.

Comment (mahalwal) [reply src/lib_ccx/params.c]: @canihavesomecoffee  ping

File: src/lib_ccx/params.c [hunk 2]
```diff
@@ -953,6 +960,22 @@ void version(char *location) {
 	mprint("	Git commit: %s\n", GIT_COMMIT);
 	mprint("	Compilation date: %s\n", COMPILE_DATE);
 	mprint("	File SHA256: %s\n", hash);
+
+	mprint("Libraries used by CCExtractor\n");
+#ifdef ENABLE_OCR
+	mprint("	Tesseract Version: %s\n", (const char*)TessVersion());
+	mprint("	Leptonica Version: %s\n", getLeptonicaVersion());
+#endif
+	mprint("	libpac Version: %s\n", GPAC_VERSION);
```
Comment (canihavesomecoffee) [inline src/lib_ccx/params.c]: it's libGPAC :)

**Pull request merged**